app/models/project.rb

```diff
@@ -1,4 +1,10 @@
 class Project < ApplicationRecord
   validates_presence_of :name, :material
   belongs_to :challenge
+  has_many :contestant_projects
+  has_many :contestants, through: :contestant_projects
+
+  def contestant_count
+    contestants.count
```

Comment:
excellent encapsulation of this calculation at the model level! I know it's a simple calculation, but it's still a calculation so to be MVC compliant, it should live here. Nice work